### ISSUE TITLE
audit needed for missing alt-text fields (e.g. when uploading images)

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#3838]
Missing Alt-text for news banner image.  There is no option to specify it when uploading the image into Janeway.  There are other instances of similar.  An audit of the whole system for missing alt text is needed, followed by updates so that such can be entered.

**Comment by StephDriver** [comment]
> * [x ]  [@ajrbyers](https://github.com/ajrbyers) to link this to core alt text implementation and close.

done while tidying up.
